### ISSUE TITLE
Broken deployment

### ISSUE THREAD

**Issue by honzajavorek** [pyvec/python.cz#416]
See https://travis-ci.org/github/pyvec/python.cz/builds/701500600, for three months we have broken deployment, so events on python.cz and/or anything else there doesn't update. There is no monitoring, so nobody noticed, until new events started to be organized and @apophys has noticed something's off in the calendar.

```
fatal: ambiguous argument 'refs/remotes/origin/gh-pages': unknown revision or path not in the working tree.

Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Usage: ghp-import [OPTIONS] DIRECTORY
ghp-import: error: Not a directory: /home/travis/build/pyvec/python.cz/pythoncz/_build
Setting up git...
Rewriting gh-pages branch...

Traceback (most recent call last):
  File "runserver.py", line 7, in <module>
    cli(app, freezer=freezer, base_url='https://python.cz')
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/elsa/_cli.py", line 158, in cli
    return command()
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/elsa/_cli.py", line 155, in deploy
    deploy_(path, remote=remote, push=push, show_err=show_git_push_stderr)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/elsa/_deployment.py", line 73, in deploy
    html_dir
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/elsa/_deployment.py", line 23, in run
    return subprocess.run(cmd, **kwargs)
  File "/opt/python/3.7.1/lib/python3.7/subprocess.py", line 481, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['ghp-import', '-n', '-m', 'Deploying :girl:', '-r', 'origin', '/home/travis/build/pyvec/python.cz/pythoncz/_build']' returned non-zero exit status 2.
```

It could possibly be a bug in [elsa](https://github.com/pyvec/elsa/) cc @hroncok, needs more investigation.

**Comment by encukou** [comment]
Looks like there's an earlier error in build, so there's nothing to upload:
```
$ pipenv run build
Generating HTML...
HEAD is now at 4312016 Add slides from Ostrava, 2020-02
Failed to fetch GitHub issues
Traceback (most recent call last):
  File "/home/travis/build/pyvec/python.cz/pythoncz/views.py", line 144, in get_involved_cs
    issues = github.get_issues(app.config['GITHUB_ORGANIZATIONS'])
  File "/home/travis/build/pyvec/python.cz/pythoncz/models/github.py", line 35, in get_issues
    issues = _sort_issues(issues)
  File "/home/travis/build/pyvec/python.cz/pythoncz/models/github.py", line 160, in _sort_issues
    return sorted(issues, key=_get_issue_sort_key, reverse=True)
  File "/home/travis/build/pyvec/python.cz/pythoncz/models/github.py", line 56, in _get_issues_for_org
    json = _request_api(session, GET_ISSUES_GRAPHQL, {'org_name': org_name})
  File "/home/travis/build/pyvec/python.cz/pythoncz/models/github.py", line 91, in _request_api
    raise requests.HTTPError(message, response=res)
requests.exceptions.HTTPError: Parse error on ")" (RPAREN) at [25, 23]
```

**Comment by honzajavorek** [comment]
There's still something wrong with it. Getting the issues from GitHub shouldn't crash the build anyway, there is a fallback. It only prints the error, but generating the output should continue.

Build: https://travis-ci.org/github/pyvec/python.cz/builds/703161401

-------

How the fallback looks like (this is the live site now, a proof that problems with fetching GitHub Issues shouldn't crash the build):

<img width="843" alt="image" src="https://user-images.githubusercontent.com/283441/86016521-270ce680-ba23-11ea-8c01-aa0be4264f59.png">

**Comment by encukou** [comment]
I see. Might it be https://github.com/pyvec/python.cz/pull/419 ?

**Comment by encukou** [comment]
Fixed by #419

**Comment by honzajavorek** [comment]
Works! https://python.cz/akce/

![](http://replygif.net/i/468.gif)
